src/status_im/transport/message/protocol.cljs

```diff
@@ -16,10 +16,11 @@
   (receive [this chat-id signature timestamp cofx] "Method producing all effects necessary for receiving the message record")
   (validate [this] "Method returning the message if it is valid or nil if it is not"))
 
-(def ^:private whisper-opts
-  {:ttl       10 ;; ttl of 10 sec
-   :powTarget config/pow-target
-   :powTime   config/pow-time})
+(def whisper-opts
+  {:whisper-drift-tolerance 10 ;; time drift that is tolerated by 10 seconds
```

Comment:
please put the comment on the previous line

Reply:
Man, C#/Go habits die hard...

Reply:
it actually can be used in clojure as well, but i encountered at least two cases when it was moved to the next line (after code re-formatting or incidentally) and so a comment was above the wrong line which is a bit misleading... And this specific thing can't happen if a comment was added before the line.